index.html

```diff
@@ -1294,6 +1449,45 @@ <h2><dfn>CaptureController</dfn>
               </li>
             </ol>
           </dd>
+
+
+          <dt><dfn>setDisplaySurfaceChangeCallback</dfn>
+          </dt>
+
+
+          <dd>
+            <p>Run the following steps:</p>
+
+
+            <ol>
+              <li>
+                <p>Let <var>callback</var> be the method's first argument.</p>
+              </li>
+
+
+              <li>
+                <p>If [=this=].{{CaptureController/[[IsBound]]}} is
+                <code>true</code>,
+                [=exception/throw=] an "{{InvalidStateError}}"
+                {{DOMException}}.</p>
+              </li>
+
+
+              <li>
+                <p>If
+                  [=this=].{{CaptureController/[[DisplaySurfaceChangeCallback]]}}
+                  is not <code>null</code>, [=exception/throw=] an
+                  "{{InvalidStateError}}" {{DOMException}}.</p>
```

Comment:
I am not sure why we should not allow to overwrite the callback either.

Reply:
This seems differ from https://github.com/w3c/mediacapture-screen-share-extensions/issues/4#issuecomment-2373957275:
> > Something like `captureController.processSourceSwitch(stream => { ... });` or `captureController.processSourceSwitch(null);`
> 
> I’m fine with this.